nucypher/policy/models.py

```diff
@@ -596,3 +600,46 @@ def by_capsule(self, capsule):
                 if saved_capsule == capsule:
                     ursulas_by_capsules[ursula] = work_order
         return ursulas_by_capsules
+
+
+class Revocation:
```

Comment:
Mmm - This is tasty as a `class`. (Still dont love the name, though).  

It... almost... feels to me like this class needs a "verb" method, but maybe it's a more static object.  Can it execute itself somehow (as well as through the Kit)?

Reply:
Hmm, I don't really like the idea of an executable `Revocation`.

The word "Revocation" is a noun, btw.

Reply:
Ah - it is a noun!

Reply:
So then - what does one instance cancel? ( "the official cancellation of a decree, decision, or promise.)

Reply:
A `Revocation` is used to revoke an `Arrangement`.

Reply:
So to be clear 1 `revocation` can cancel 1 `arrangement`?

Reply:
Yep, it includes the `arrangement_id`.

Reply:
So then why not give it a method to do so?

Reply:
I guess you could, but I'm not sure if that's the best place to do it. We probably need to talk about if that's something we want to support there.

Reply:
Well - without it - the only way to interface with revoke is with the kit, right?

Reply:
As of now, this only supports Alice's ability to revoke. What you're suggesting is a departure from that and might require some further work to allow _anyone_ to revoke as long as they have the signed revocation.

Reply:
Oh - I understood that to be the concept.  What additional work is required?

Reply:
To answer your question, the correct way to revoke is to have Alice call `revoke` with the specific `Policy`.

Reply:
Yeah - I see - interesting.   Well - I think it's worth the extra leg work :-)

Reply:
Is that something we want to do _now_, though? I'm okay with doing it, it's not much extra. But it does represent a large feature addition to allow Alice have _anyone_ revoke when she simply gives away this signed revocation.

Reply:
Need some team input here.

Reply:
Tasty or not, can we get a docstring?